docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -16,6 +16,30 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hms-virtual-background      | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-virtual-background)           |
 
 
+## 2024-11-11
+
+| Package                           | Version |
+| --------------------------------- | ------- |
+| @100mslive/hms-video-store        | 0.12.24 |
+| @100mslive/react-sdk              | 0.10.24 |
+| @100mslive/roomkit-react          | 0.3.24  |
+| @100mslive/hms-whiteboard         | 0.0.14  |
+| @100mslive/hls-player             | 0.3.24  |
+| @100mslive/hms-noise-cancellation | 0.0.1   |
+| @100mslive/hms-virtual-background | 1.13.24 |
+
+### Added
+
+- Added a new field - `track_type` - in `HMSException` when there is a Track exception
+- In Roomkit Prebuilt, now on Mobile Browsers, Stats for Nerds is available to be viewed
+
+### Fixed
+
+- Tiles going out of view in recording for 480p resolution when there are more than 3 peers
+- Roomkit Prebuilt: Previous session's device selection not being preserved when Skip Preview was enabled
+- Roomkit Prebuilt: Peer stuck on loading screen after staying in preview for 15 mins instead of being redirected to the rejoin page
```

Comment:
```suggestion
- Roomkit Prebuilt: Peer stuck on loading screen after staying in preview for 15 mins instead of being redirected to the leave page
```